client/landing/site.landing/coffee/team/forms/teamdomaintabform.coffee

```diff
@@ -19,42 +19,34 @@ module.exports = class TeamDomainTab extends kd.FormView
       click        : => @input.setFocus()
 
     @inputView.addSubView @input = new kd.InputView
-      placeholder  : 'your-team'
-      attributes   : 10
+      placeholder  : 'Your team name...'
```

Comment:
Initially `attributes` was intended to set `size` of input - https://github.com/koding/koding/commit/a439a14875acfa2442949e938f8295f43f35c98a#diff-0c95950b99617e59667bc6440ea09ac1. I guess it was needed to meet requirements of old design. However, later it was changed just to `attributes: 10` https://github.com/koding/koding/commit/85661dbc6fae57444242c8b48b049cd9c5cc478d probably by mistake. I don't think that strange value affected input somehow that's why I decided to remove it.